pulpcore/cli/deb/context.py

```diff
@@ -75,12 +75,25 @@ class PulpAptRemoteContext(PulpEntityContext):
     CREATE_ID = "remotes_deb_apt_create"
     UPDATE_ID = "remotes_deb_apt_partial_update"
     DELETE_ID = "remotes_deb_apt_delete"
+    NULLABLES = {"architectures", "components"}
+
+    def tuple_to_whitespace_separated_string(field_name: str, body: EntityDefinition) -> None:
+        """
+        Adjusts the field field_name within the body in place.
+        Tuple-lists are turned into whitespace separated strings.
+        """
+        field = body.pop(field_name, None)
+        if field is not None:
+            if not isinstance(field, tuple):
+                # Is this exception condition needed?! I don't think it can not be a tuple...
+                raise Exception
```

Comment:
Like the comment says, I am not sure if this is really needed. Will perform some targeted testing.

Reply:
You should be careful here. Despite distributions, components and architectures may be `None` deliberately to clear them on the server.